mobile/src/package.json

```diff
@@ -18,7 +18,7 @@
     "axios": "^0.20.0",
     "expo": "^38.0.10",
     "expo-constants": "~9.1.1",
-    "expo-notifications": "~0.3.3",
+    "expo-notifications": "^0.3.3",
```

Comment:
The app didn't run locally until I ran `npm install expo-notifications`. Is it okay the changes from that in this PR?

Reply:
@shashankanandkannan why didn't you do `npm install` instead? Did that not work? You can also discard the changes made to package.json and package-lock.json if they aren't necessary

Reply:
I did, but it didn't wor for some reason. I got an error saying it couldn't find expo-notifications, so I specifically installed it.

Reply:
oh wait, I think you need to do ```expo install``` instead

Reply:
I had the same error and did `npm i` and things worked fine for me

Reply:
not sure why it didnt fix your issue